lang/luajit/Makefile

```diff
@@ -0,0 +1,91 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Does the copyright apply to the Makefile only? I mean I'm reserving the rights for the package or the Makefile?